docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LightGbmAdvanced.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic.Trainers.Regression
+{
+    class LightGbmAdvanced
```

Comment:
No. It's for pipeline we don't quite care in API docs. But I feel it will be useful in the future.

---
In reply to: [275456651](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275456651) [](ancestors = 275456651)